src/search/pdbs/pattern_collection_generator_single_cegar.cc

```diff
@@ -56,7 +56,28 @@ static shared_ptr<PatternCollectionGenerator> _parse(
             "AAAI Press",
             "2019"));
     add_implementation_notes_to_parser(parser);
-    add_cegar_options_to_parser(parser);
+    parser.add_option<int>(
```

Comment:
Could you add a TODO and an issue to move these options to the base class of pattern generators? I think we really want them everywhere.